content/en/news/kubesphere-geko-partnership.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Embrace KubeSphere Spanish Community: Geko and KubeSphere Build Partnership'
+createTime: '2020-07-22'
+author: 'Feynman, Sherlock'
+---
+
+July 22, 2020 - KubeSphere and Geko are pleased to announce a joint partnership to deliver outstanding value to KubeSphere users in Spain, and further to Europe. As part of KubeSphere’s endeavor to bring the fruitful result of open source to users around the world, this marks a great milestone for the container platform as we are embracing a more diversified and vibrant community.
+
+Spain is home to one of the most active open source communities in Europe. Geko, a strong and active player in the cloud industry of Spain, offers state-of-the-art technologies that help companies with cloud adoption, hybrid computing, container artifact development, and DevOps. In this connection, Geko has made great contribution to cloud native technologies, which echoes KubeSphere’s commitment to open source, with a shared aspiration to benefit everyone.
+
+Ray Zhou, head of KubeSphere project, highlighted KubeSphere’s decision to work with Geko. “KubeSphere is designed for users around the world at the very beginning. Our partnership with Geko provides us with an exciting opportunity to further empower innovators in the Spanish community that is leading the country’s technology revolution. We hope that more Spanish developers can be motivated to join us in different aspects so that the fruit of open source can be shared by everyone.”
+
+By building this strong partnership, KubeSphere and Geko look to better empower both community members as they are connected to exchange innovative solutions and explore cutting-edge technologies. Under the partnership, Geko will start by localizing the web console of KubeSphere v3.0, which will be released very soon. Local websites, documentation and more will be coming as the local community grows. Geko will also hold various activities to engage community members going forward, including workshops and meetups.
+
+The partnership will take effect from July 2020 and can be extended one year later for further cooperation.
+
+## About Geko
+
+Our skills are focused on cloud platforms and our expertise with Internet projects bring us a lot of resources with projects such as cloud migration, DRS, resilience architecture, full managed service, green cloud computing and cost efficiency.
```

Comment:
I think we should introduce Geko in the third person.

Reply:
I copied this part from Geko official website, but I made the change any way.